src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -30,35 +31,33 @@
 
         return result;
     };
-    const getInvalidTabs = (validator) => {
-        return validator.fieldsToValidate.reduce((invalidTabs, field) => {
-            const tabPane = field.item.closest('.tab-pane');
+    const getInvalidSections = (validator) => {
+        return validator.fieldsToValidate.reduce((invalidSections, field) => {
+            const section = field.item.closest('.ibexa-anchor-navigation-sections__section');
 
-            if (tabPane && field.item.classList.contains('is-invalid')) {
-                invalidTabs.add(tabPane.id);
+            if (section && field.item.classList.contains('is-invalid')) {
+                invalidSections.add(section.dataset.anchorSectionId);
             }
 
-            return invalidTabs;
+            return invalidSections;
         }, new Set());
     };
     const fields = doc.querySelectorAll('.ez-field-edit');
     const focusOnFirstError = () => {
         const invalidFields = doc.querySelectorAll('.ez-field-edit.is-invalid');
+        const invalidSection = invalidFields[0].closest('.ibexa-anchor-navigation-sections__section');
 
         fields.forEach((field) => field.removeAttribute('tabindex'));
         invalidFields.forEach((field) => field.setAttribute('tabindex', '-1'));
 
-        invalidTab = invalidFields[0].closest('.tab-pane');
+        if (invalidSection) {
+            const { anchorSectionId } = invalidSection.dataset;
+            const invalidButton = doc.querySelector(`[data-anchor-target-section="${anchorSectionId}"`);
 
-        if (invalidTab) {
-            const invalidTabLink = doc.querySelector(`a[href="#${invalidTab.id}"]`);
-
-            invalidTabLink.click();
+            invalidButton.click();
```

Comment:
Do we click it to basically call `navigateTo` function from `admin.anchor.navigation.js`?